CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact docs-utils module, newest changes on the top.
 
+## [unreleased]
+
+### ⚠ BREAKING CHANGE
+
+- Require node 14 or later from 0.4.0 version
```

Comment:
We normally write change logs in the past tense.

```suggestion
- Changed to require node 14 or later
```
Also, would you please update "Update" to "Updated" from line 13?